nebari-dev/nebari
# [ENH] - Include "--attempt-fixes" flag from Nebari upgrade CLI in upgrade steps logic

## Reference Issues or PRs

This resolves #2761 by passing the `attempt_fixes` variable from the command line option to the upgrade methods.
close #2761


## What does this implement/fix?

_Put a `x` in the boxes that apply_


## Testing


## How to test this PR?

Run `nebari upgrade --attempt-fixes` and verify that the upgrade is successful without prompts from the user.



## Any other comments?
